### PR TITLE
X-Ray Tracing and other changes to Java examples

### PR DESCRIPTION
**Pull request by muraria**
*Issue #, if available:53

* Added Tracing both example
* Removed LocalStack from DDB Example
* Remove Region dependency from DDB example
* Added Tired Compilation stop at Level 1 to DDB example

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by dmahapatro** [comment]
@muraria Can we also add a snapshot of the trace view which response time from each service?

**Comment by muraria** [comment]
> @muraria Can we also add a snapshot of the trace view which response time from each service?

Updated the image

File: java-test-samples/apigw-lambda-list-s3-buckets/README.md [hunk 1]
```diff
@@ -228,8 +228,11 @@ TODO
 
 ## Implement application tracing
 You can use AWS X-Ray to track user requests as they travel through your entire application. With X-Ray, you can understand how your application and its underlying services are performing to identify and troubleshoot the root cause of performance issues and errors.
+This [Lambda function](./src/main/java/com/example/App.java) handler has been instrumented using AWS X-Ray. Find your Lambda function in the console. Then navigate to Monitor -> Traces and you should see a graph of your X-Ray trace.
```
Comment (dmahapatro) [inline java-test-samples/apigw-lambda-list-s3-buckets/README.md:231]: Is it worth mentioning that when running tests at scale, x-ray cost will be incurred?

Comment (muraria) [reply java-test-samples/apigw-lambda-list-s3-buckets/README.md:231]: Probably. But, if we are adding that note, I think we should be adding it to other language samples as well to keep it consistent.

**Review by dmahapatro — APPROVED** [review]

**Pull request merged**